pkg/scheduler/framework/session_plugins.go

```diff
@@ -198,6 +229,24 @@ func (ssn *Session) JobReady(obj interface{}) bool {
 	return true
 }
 
+// BackFillEligible check backfill eligiblity
+func (ssn *Session) BackFillEligible(obj interface{}) bool {
+	for _, tier := range ssn.Tiers {
+		for _, plugin := range tier.Plugins {
+			jrf, found := ssn.backFillEligibleFns[plugin.Name]
```

Comment:
should be also configurable

Reply:
"configurable" as in "can be configured to turn on and off"?

Reply:
yes.

Reply:
Backfill function can be enabled/disabled from configurations. `BackfillEligible` is a function provided by each plugin to decide which job is eligible to be a backfill job. What is the use case to conditionally turn on/off `BackfillEligible` function?

Reply:
> What is the use case to conditionally turn on/off BackfillEligible function?

Different plugin may provide this callback, we need a configuration to controll it.